minecraft-server/start-deployFabric

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+set -u
+
+export TYPE=FABRIC
+
+FABRIC_INSTALLER=${FABRIC_INSTALLER:-}
+FABRIC_INSTALLER_URL=${FABRIC_INSTALLER_URL:-}
+FABRICVERSION=${FABRICVERSION:-LATEST}
+if [[ -z $FABRIC_INSTALLER && -z $FABRIC_INSTALLER_URL ]]; then
+  echo "Checking Fabric version information."
+  case $FABRICVERSION in
+    LATEST)
+      FABRIC_VERSION=$(python - << 'EOF'
```

Comment:
I was considering removing python from the image, but the alternatives, such as

https://github.com/jeffbr13/xq

would require python based tools :). So you've given me a good reason to simply keep python included.

Reply:
I'm not a huge fan of this whole "inline python script into bash" mess, but it also seemed to be the cleanest way to parse XML without having to introduce anything new into the base image.

If you're okay with adding xq into the base image, then I'll rewrite the deploy script to take advantage of it so it's a little easier to read (and also follows the same pattern in the forge deploy scripts of `curl FOO | jq BAR`)

Reply:
Yeah, give the xq approach a try especially since like you said, it follows the pattern of the jq usage.